### PR TITLE
Fix handling of extra assets

### PR DESCRIPTION
**Pull request by stbenjam**
TRT-676

I'm trying to use the extra assets feature in my PR[1] but it has two problems:

  - it errors if the directory is empty
  - it does not look for .yml files (only .yaml)

This fixes both problems.

[1] https://github.com/openshift/release/pull/35105

### CODE REVIEW

File: utils.sh [hunk 1]
```diff
@@ -137,7 +137,7 @@ function create_cluster() {
     fi
 
     if [ ! -z "${ASSETS_EXTRA_FOLDER:-}" ]; then
-      cp -rf ${ASSETS_EXTRA_FOLDER}/*.yaml ${assets_dir}/openshift/
+      find "${ASSETS_EXTRA_FOLDER}" \( -name \*.yml -or -name \*.yaml \) -exec cp {} "./${assets_dir}/openshift" \;
```
Comment (derekhiggins) [inline utils.sh:140]: I'm not sure if anybody actually sets OCP_DIR but if there do and its an absolute directory is will find its way to $assets_dir , which wont work here with the "./" prefixed

Comment (stbenjam) [reply utils.sh]: Makes sense, fixed.  Not sure why I added it to begin with.

**Comment by stbenjam** [comment]
Needs a lgtm as well please @derekhiggins @bfournie

**Pull request merged**